addons/web/static/tests/chrome/keyboard_navigation_mixin_tests.js

```diff
@@ -50,7 +55,12 @@ QUnit.module('KeyboardNavigationMixin', function () {
                 this._super.apply(this, arguments);
                 KeyboardNavigationMixin.init.call(this);
             },
+            destroy: function () {
+                KeyboardNavigationMixin.destroy.call(this);
+                return this._super(...arguments);
+            },
             start: function () {
+                KeyboardNavigationMixin.start.call(this);
```

Comment:
Same comment as FJA for all files :+1:

Reply:
None of the methods on `AbstractWebClient` have docstrings, should I add them everywhere or just those I touched?

Reply:
Just what you touch. But here it is mainly the order start -> destroy that is needed

Reply:
Should be good now :)